linera-core/src/client.rs

```diff
@@ -1294,6 +1361,7 @@ where
         Ok(certificate)
     }
 
+    #[tracing::instrument(level = "trace", skip(block))]
```

Comment:
Optional: maybe record the block?

```suggestion
    #[tracing::instrument(level = "trace")]
```